src/main/java/org/openstreetmap/atlas/checks/atlas/predicates/ValuePredicates.java

```diff
@@ -1,56 +1,47 @@
 package org.openstreetmap.atlas.checks.atlas.predicates;
 
-import static org.openstreetmap.atlas.checks.constants.CommonConstants.CLOSED_PARENTHESES_CHAR;
```

Comment:
Was sonar complaining about using these? If not, we can still leverage these characters

Reply:
Sonar was complaining about the usage of Stack and also a line inside the logic. So, I modified the logic.